practicals/Recommender_Systems.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Could you please make the distinction between collaborative and content-based filtering a bit more clear?</p><p> </p><p>"users" -&gt; user's</p><p> </p><p>I think the matrix factorisation could be made a bit easier here by walking them through it a bit more. For example: set up example U, V matrices, show what one dot product would be and how it corresponds to a movie rating, and then computing the matrix product and show how it aligns (or almost aligns) with a feedback matrix.</p><p>At the moment, it feels like a little too much theory has been thrown at them all at once.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>